src/sbt-test/sbt-reactive-app/native-packager-cross-version-compat/test

```diff
@@ -0,0 +1,7 @@
+# This test only checks `sbt` starts successfully into a shell.
+# Since Play 2.6.21 there's a dependency issue between Play's sbt-plugin and sbt-reactive-app where both bring in
+# incompatible versions of `sbt-native-packager`. Users adding both `sbt-reactive-app` and a direct or transitive
+# dependency to an incompatible version of `sbt-reactive-app` would be DoS-ing themselves.
```

Comment:
You said sbt-reactive-app twice.

Also, DoS-ing themselves?  Sounds extreme.  For a second I thought you meant to say BSoD :)

Reply:
🤦‍♂️

Reply:
Fixed the comment. Left `Main` and `build.sbt`

Reply:
Actually, I'm quite intrigued to see if removing `build.sbt` would reproduce the error. So there'd still be `plugins.sbt` adding the two clashing plugins but no project would be explictly adding `SbtReactiveAppPlugin` so maybe the issue won't happen? 
I can leave without the answer.

Reply:
You're right.  `SbtReactiveAppPlugin` doesn't auto-trigger, so the minimal build to trigger this should be `enablePlugins(SbtReactiveAppPlugin)`.